modules/installation-osp-kuryr-config-yaml.adoc

```diff
@@ -53,6 +53,10 @@ platform:
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed25519 AAAA...
 ----
+[NOTE]
+====
+Be aware that CIDR sizes for namespaces are not modified post-deployment. Red Hat does not provide direct guidance on sizing your OpenShift Container Platform cluster. Determining the size within the supported bounds requires careful consideration of all the multidimensional factors that limit the cluster scale.
```

Comment:
Shouldn't it be "CIDR sizes for namespaces are not adjustable post-deployment"?

Reply:
You are right @dulek ! I have made the update!